### PR TITLE
Add heap part js code

### PR DESCRIPTION
**Pull request by what-is-me**
If this PR is related to coding or code translation, please read the [contribution guideline](https://github.com/krahets/hello-algo/issues/15#issue-1464069591), fill out the checklist, and paste the console outputs to the PR.

- [ ] I've tested the code and ensured the outputs are the same as the outputs of reference codes.
- [ ] I've checked the codes (formatting, comments, indentation, file header, etc) carefully.
- [ ] The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

```

输入列表并建堆后
堆的数组表示：    
[
  9, 8, 6, 6, 7,  
  5, 2, 1, 4, 3,  
  6, 2
]
堆的树状表示：    
        /——— 2    
    /——— 6        
   |    \——— 5    
   |        \——— 2
——— 9
   |        /——— 6
   |    /——— 7    
   |   |    \——— 3
    \——— 8        
       |    /——— 4
        \——— 6    
            \——— 1

堆顶元素为 9

元素 7 入堆后
堆的数组表示：
[
  9, 8, 7, 6, 7, 6,
  2, 1, 4, 3, 6, 2,
  5
]
堆的树状表示：
        /——— 2
    /——— 7
   |   |    /——— 5
   |    \——— 6
   |        \——— 2
——— 9
   |        /——— 6
   |    /——— 7
   |   |    \——— 3
    \——— 8
       |    /——— 4
        \——— 6
            \——— 1

堆顶元素 9 出堆后
堆的数组表示：
[
  8, 7, 7, 6, 6,
  6, 2, 1, 4, 3,
  5, 2
]
堆的树状表示：
        /——— 2
    /——— 7
   |    \——— 6
   |        \——— 2
——— 8
   |        /——— 5
   |    /——— 6
   |   |    \——— 3
    \——— 7
       |    /——— 4
        \——— 6
            \——— 1

堆元素数量为 12

堆是否为空 false
```
js没有内置堆类

### CODE REVIEW

**Comment by krahets** [comment]
Pending for review by @justin-tse

**Review by justin-tse — COMMENTED** [review]
@what-is-me Thanks! Please address the comments.

File: codes/javascript/include/PrintUtil.js [hunk 1]
```diff
@@ -82,7 +84,19 @@ function showTrunks(p) {
     process.stdout.write(p.str);
 }
 
+/**
+ * Print a heap
+ * @param {number[]} arr
```
Comment (justin-tse) [inline codes/javascript/include/PrintUtil.js:89]:
```suggestion
 * @param arr
```

**Review by justin-tse — APPROVED** [review]
Awesome! Thanks @what-is-me

**Review by krahets — APPROVED** [review]

**Pull request merged**